modules/collector-source/pkg/collector/info.go

```diff
@@ -0,0 +1,30 @@
+package collector
+
+// InfoAggregator is metric aggregator meant to just record label values
+type InfoAggregator struct {
+	name        string
+	labelValues []string
+}
+
+func Info(name string, labelValues []string) MetricAggregator {
+	return &InfoAggregator{
+		name:        name,
+		labelValues: labelValues,
+	}
+}
+
+func (m *InfoAggregator) Name() string {
+	return m.name
+}
+
+func (m *InfoAggregator) LabelValues() []string {
+	return m.labelValues
+}
+
+func (m *InfoAggregator) Update(value float64) {
+
+}
+
+func (m *InfoAggregator) Value() float64 {
+	return 0
```

Comment:
I think most of these return `1` in prom. We can do the same for equivalence. There may be some underlying reason /shrug